client/llb/fileop.go

```diff
@@ -321,9 +327,13 @@ type fileActionMkfile struct {
 }
 
 func (a *fileActionMkfile) toProtoAction(ctx context.Context, parent string, base pb.InputIndex) (pb.IsFileAction, error) {
+	normalizedPath, err := normalizePath(parent, a.file, false)
```

Comment:
Since you are repeating the normalization of the path over and over would it make sense to move the error handler as well to the function rather than copy and pasting the code?